src/com/content-games/games.js

```diff
@@ -152,14 +156,28 @@ export default class ContentGames extends Component {
 				return <ContentItemBox node={r.node} user={props.user} path={props.path} noevent={props.noevent ? props.noevent : null} />;
 			});
 
+			while (Games.length % layout !== 0) {
```

Comment:
Could I get a comment here (line above) describing the cleverness going on (i.e. integer modulus is not a very common operation)?

Reply:
Sure thing. Basically checks if the number of items evenly divides with number of columns

Reply:
Sounds good.